project.inlang.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please remove this file

Reply:
@ptdatta are you going to finish this?

Reply:
I thought it's already completed.

Reply:
@ptdatta you thing that you have removed this file?